src/util/DOMHelper.js

```diff
@@ -33,6 +35,20 @@ export function getClosestParentCustomElementNode(domNode) {
 	return customElementParents.pop();
 }
 
+export function getAllElementChildren(domNode) {
```

Comment:
@quarkus If we wanted to avoid the circular dependecy we could add a paramter for the second check. Kind of the same as in the node iterator. You could pass a filter function from outside where you could make the instanceof check.

Reply:
@eddyloewen i think i did the change :) .. can you please review again ?
Thanks